src/transformers/modeling_utils.py

```diff
@@ -3748,11 +3748,13 @@ def _fix_key(key):
                 else:
                     _loaded_keys = loaded_keys
                 not_initialized_submodules = set_initialized_submodules(model, _loaded_keys)
-                # if we're about to tie the output embeds to the input embeds we don't need to init them
+                # If we're about to tie the output embeds to the input embeds we don't need to init them
                 if hasattr(model.config, "tie_word_embeddings") and model.config.tie_word_embeddings:
                     output_embeddings = model.get_output_embeddings()
                     if output_embeddings is not None:
-                        output_embeddings._is_hf_initialized = True
+                        # Still need to initialize if there is a bias term since biases are not tied.
+                        if not hasattr(output_embeddings, "bias") or output_embeddings.bias is None:
```

Comment:
More context for this change here:
https://github.com/huggingface/transformers/pull/28192#issuecomment-1934913583

The test for ElectraModel was failing because of this. For some reason, the test only ran after I modified the source for ElectraModel

Reply:
I pulled this change out to its own PR:
https://github.com/huggingface/transformers/pull/28947

Once that goes in, I'll remove it from this PR.